website/content/en/blog/kubecon-eu-2025.md

```diff
@@ -0,0 +1,77 @@
+
+### Wednesday, April 2nd: Conference Day 1
+| Time | Location | Topic | Presenter |
+|-----|----------|-------|-----------|
+| 07:15 - 08:45 | [Costa Coffee](https://www.excel.london/visitor/food-drink/costa-coffee) , East Entrace, Excel, London | Platform Coffee Meetup | CNCF Platforms Working Group |
+| 10:30 - 12:00 | Booth TAG-A | Open Booth | TAG Leads |
+| 12:00 - 12:10 | Booth TAG-A | Lightning Talk - "Lightning-Fast Local AI App Development with Podman" | [Daniel Oh](https://www.linkedin.com/in/daniel-oh-083818112/) | 12:15 - 12:25 | Booth TAG-A | Lightning Talk - "Build Your Perfect App Delivery Model: Customizing Standards with Crossplane XRDs" | Sarah Murphy |
+| 12:30 - 12:40 | Booth TAG-A  | Lightning Talk - "Platform Engineering untold thruths: is it just an infrastructure matter?" | [Graziano Casto](https://www.linkedin.com/in/castograziano/) |
+| 12:45- 12:55 | Booth TAG-A  | Lightning Talk - "Platformization: Perceiving Platform Patterns" | Ram Iyengar |
+| 1:00 - 2:30 | Booth TAG-A | Platforms Working Group | WG Leads |
+| 2:30- 4:00 | Booth TAG-A | App Development Working Group | WG Leads |
+| 4:00 - 6:00 | Booth TAG-A | Open Booth | TAG Leads |
+| 6:00- 8:00 | Booth TAG-A | KubeCrawl | TAG Leads |
+
+
+
+### Thursday, April 3rd: Conference Day 2
+| Time | Location | Topic | Presenter |
+|-----|----------|-------|-----------|
+| 07:15 - 08:45 | [Costa Coffee](https://www.excel.london/visitor/food-drink/costa-coffee) , East Entrace, Excel, London | Platform Coffee Meetup | CNCF Platforms Working Group |
+| 10:30 - 12:00 | Booth TAG-A | Open Booth | TAG Leads |
+| 12:00 - 12:10 | Booth TAG-A | Lightning Talk - "Scale Before It Hurts: KEDA for Data-Intensive Apps" | [Graziano Casto](https://www.linkedin.com/in/castograziano/) & [Alessandro Cannarella](https://www.linkedin.com/in/alessandro-cannarella/) |
+| 12:15 - 12:25 | Booth TAG-A  | Lightning Talk - "Extensibility and Portability with Radius for Platform Engineers" | Johnathan Smith |
+| 12:30 - 12:40 | Booth TAG-A  | Lightning Talk - "Is there a tool for this? - App Developer Edition" | Alexandra Aldershaab |
+| 12:45 - 12:55 | Booth TAG-A  | Lightning Talk - "Platformization: Perceiving Platform Patterns" | Ram Iyengar |
+| 1:00 - 2:30 | Booth TAG-A | Infrastructure Lifecycle Working Group | WG Leads |
+| 2:30- 4:00 | Booth TAG-A | Artifacts Working Group | WG Leads |
+| 4:00 - 5:00 | Booth TAG-A | Open Booth | TAG Leads |
+
+### Friday, April 4th: Conference Day 3
+| Time | Location | Topic | Presenter |
+|-----|----------|-------|-----------|
+| 07:15 - 08:45 | [Costa Coffee](https://www.excel.london/visitor/food-drink/costa-coffee) , East Entrace, Excel, London | Platform Coffee Meetup | CNCF Platforms Working Group |
+| 10:30 - 12:00 | Booth TAG-A | Open Booth | TAG Leads |
+| 12:00 - 12:10 | Booth TAG-A  | Lightning Talk - "Your Application Delivery Ran Over My CI/CD: Expanding Beyond Pipeline Thinking" | Luke Philips |
```

Comment:
Can Time be set in 24HH format? 

Also, if possible can my talk (Luke) be moved to Thursday? My Fri afternoon schedule will be tight

Reply:
@lukepatrick I'll adjust the schedule shortly